### PR TITLE
[release] merging master-release-update into master

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin @sdroege

### CODE REVIEW

File: gtk-sys/Cargo.toml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [package]
 name = "gtk-sys"
-version = "0.0.1"
+version = "0.0.2"
```
Comment (EPashkin) [inline gtk-sys/Cargo.toml]: Strange version

Comment (EPashkin) [reply gtk-sys/Cargo.toml]: Was broken in https://github.com/gtk-rs/sys/commit/8b30f8692b82bbc07273460c21c9157897dcdc82

Comment (GuillaumeGomez) [reply gtk-sys/Cargo.toml]: Excellent catch! Fixed!

**Comment by EPashkin** [comment]
👍

**Pull request merged**